builtin/providers/aws/resource_aws_api_gateway_deployment.go

```diff
@@ -0,0 +1,136 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/apigateway"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsApiGatewayDeployment() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsApiGatewayDeploymentCreate,
+		Read:   resourceAwsApiGatewayDeploymentRead,
+		Delete: resourceAwsApiGatewayDeploymentDelete,
+
+		Schema: map[string]*schema.Schema{
+			"api_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"stage_name": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
+
+			"description": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
```

Comment:
I think [_it is possible_ to update Deployments](http://docs.aws.amazon.com/apigateway/api-reference/link-relation/deployment-update/) description, so it shouldn't be `ForceNew` + an Update function will need to be implemented for this resource.